lib/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-cps-parent</artifactId>
+      <version>${changelist}</version>
+  </parent>
+
+  <groupId>com.cloudbees</groupId>
```

Comment:
Should we switch the `groupId` for `groovy-cps` to `org.jenkins-ci.plugins.workflow` going forward? I don't think it really matters either way.

Reply:
I would leave the `groupId` alone unless you are sure there would be no consequences to changing it.

Reply:
Ok, looks like we agree. I will leave it.